README.md

```diff
@@ -101,11 +101,23 @@ sentry_upload_proguard(
   auth_token: '...', # Do not use if using api_key
   org_slug: '...',
   project_slug: '...',
-  android_manifest_path: 'path to merged AndroidManifest file' # found in `app/build/intermediates/manifests/full`
+  android_manifest_path: 'path to merged AndroidManifest file', # found in `app/build/intermediates/manifests/full`
   mapping_path: 'path to mapping.txt to upload',
 )
 ```
 
+#### Upload Debugging Information Files
+
+```ruby
+sentry_upload_dif(
+  api_key: '...', # Do not use if using auth_token
+  auth_token: '...', # Do not use if using api_key
+  org_slug: '...',
+  project_slug: '...',
+  path: '/path/to/files'
```

Comment:
Is it possible to make this default to `.`?  IIUC this is mostly what does the right thing, see e.g. https://github.com/getsentry/sentry-cocoa/blob/ci/app-store-connect/fastlane/Fastfile#L93-L101

Reply:
@denrase, I think this should be an easy fix ⬆️ . Can you maybe open up a PR for that?

Reply:
@philipphofmann  PR is ready for review: https://github.com/getsentry/sentry-fastlane-plugin/pull/94